crates/bevy_math/src/cubic_splines.rs

```diff
@@ -913,4 +1173,42 @@ mod tests {
         assert!(bezier.ease(0.5) < -0.5);
         assert_eq!(bezier.ease(1.0), 1.0);
     }
+
+    #[test]
```

Comment:
This is a really good test. Ideally, I would like at-least one more using comparing points on a nonuniform, weighted, 3d curve against "ground-truth" data computed using tinyspline (with appropriate deltas to accommodate floating point error).

Reply:
Yeah I agree a ground-truth test is also needed, but I felt that was over the scope of what I was doing.

Reply:
You can either add it yourself when this gets merged, or if you have a test-case in mind, send me the data and I'll put it in.

Reply:
Yep, just noting what I'm sure Alice will want. We should totally merge this into the PR first.